index.js

```diff
@@ -1,27 +1,41 @@
-/* globals require, __dirname, process */
-
 const path = require('path');
+const builder = require('adguard-hostlists-builder');
 const fs = require('fs');
-const builder = require("adguard-hostlists-builder");
+const { checkRemovedServices } = require('./scripts/services/check-removed-services');
+const { rewriteServicesJSON } = require('./scripts/services/rewrite-services-json');
+
+const filtersDir = path.join(__dirname, './filters');
+const assetsDir = path.join(__dirname, './assets');
+const tagsDir = path.join(__dirname, './tags');
+const localesDir = path.join(__dirname, './locales');
+const inputServicesDir = path.join(__dirname, './services');
+const outputServicesFile = path.join(assetsDir, 'services.json');
+
+// build services.json from services folder
+const buildServices = async (dirPath, filePath) => {
+    try {
+        await checkRemovedServices(dirPath, filePath);
+        await rewriteServicesJSON(dirPath, filePath);
+        console.log('Successfully finished building services.json');
+        process.exit(0);
+    } catch (error) {
+        console.log('Building services.json finished with an error', error);
+        process.exit(1);
+    }
+};
 
 // Validate services.json and make sure it is a valid JSON.
```

Comment:
this better to be inside some function which will be called in the entry point

Reply:
can this be done at the beginning of buildServices? e.g. 
```
const buildServices = async (dirPath, filePath) => {
    try {
        validateJSON(filePath);
        await checkRemovedServices(dirPath, filePath);
        ....
}
```

Reply:
yes